CircleCI failed because of missing dependencies. When are we supposed to fix it? I forgot if we were to ignore CircleCI for now or not.

As for Travis CI, GitHub blocked it for "API limits exceeded", but when I tested it locally, I get this:
```
>           raise AssertionError(os.linesep.join(other_warns))
E           AssertionError: Working outside of application context.
E           
E           This typically means that you attempted to use functionality that needed
E           to interface with the current application object in a way.  To solve
E           this set up an application context with app.app_context().  See the
E           documentation for more information.

changebot/github/tests/test_github_api.py:77: AssertionError
```
I think this other error is real and somehow related to #84? @Cadair ?

--- a/changebot/github/github_api.py
+++ b/changebot/github/github_api.py
@@ -7,7 +7,6 @@ from datetime import datetime, timedelta
 
 import dateutil.parser
 import toml
-from flask import current_app
 
 from changebot.github.github_auth import github_request_headers
 
@@ -133,7 +132,7 @@ class RepoHandler(object):
         try:
             file_content = self.get_file_contents(path_to_file)
             cfg = toml.loads(file_content)
-            cfg = cfg['tool'][current_app.bot_username]
+            cfg = cfg['tool']['astropy-bot']
         except Exception as e:
             if warn_on_failure:
                 warnings.warn(str(e))

--- a/changebot/github/tests/test_github_api.py
+++ b/changebot/github/tests/test_github_api.py
@@ -50,7 +50,6 @@ class TestRepoHandler:
 # NOTE: Might hit API limit?
 class TestRealRepoHandler:
     def setup_class(self):
-        # TODO: Use astropy/astropy-bot when #42 is merged.
         self.repo = RepoHandler('astropy/astropy-bot')
 
     def test_get_config(self):
@@ -66,7 +65,7 @@ class TestRealRepoHandler:
         other_warns = []
         for ww in w:
             s = str(ww.message)
-            if 'API limit' in s:
+            if 'API rate limit' in s:
                 hit_api_limit = True
             else:
                 other_warns.append(s)

